saleor/graphql/webhook/subscription_types.py

```diff
@@ -1077,6 +1103,123 @@ def resolve_warehouse(root, _info):
         return warehouse
 
 
+class PaymentBase(AbstractType):
+    payment = graphene.Field(
+        "saleor.graphql.payment.types.Payment",
+        description="Look up a payment." + ADDED_IN_35 + PREVIEW_FEATURE,
```

Comment:
I know that we have a mess in the rest of the events but I don't see any reason why we couldn't clean it up for new events. I mean:
`ADDED_IN_35 + PREVIEW_FEATURE` (Should be 36), should be added on each event lvl, not on a field lvl. 
Base shouldn't have ANY labels like `ADDED_XX` or `PREVIEW_FEATURE` as this type is commonly shared between different events that can be introduced in a different version of Saleor.

Reply:
fixed it of new events